interactions/client/context.py

```diff
@@ -63,6 +63,13 @@ class _Context(ClientSerializerMixin):
     locale: Optional[Locale] = field(converter=Locale, default=None)
     guild_locale: Optional[Locale] = field(converter=Locale, default=None)
 
+    @property
+    def deferred_ephemeral(self) -> bool:
+        return bool(
```

Comment:
```suggestion
        return (
```

No unnecessary casts to bool :)

Reply:
I tried without the bool it returns the enum
```
b4 adding bool
INFO:root:MessageFlags.LOADING
after adding bool
INFO:root:True
```
My logs when b4 I add bool and after add bool

Reply:
That's interesting, bool can stay then

(However the Enum would still be true if its not 0)